pybliometrics/scopus/abstract_retrieval.py

```diff
@@ -731,15 +741,16 @@ def __init__(self,
         self._refresh = refresh
         Retrieval.__init__(self, identifier=identifier, id_type=id_type,
                            api='AbstractRetrieval', **kwds)
-        self._json = self._json['abstracts-retrieval-response']
-        self._head = chained_get(self._json, ["item", "bibrecord", "head"], {})
-        conf_path = ['source', 'additional-srcinfo', 'conferenceinfo', 'confevent']
-        self._confevent = chained_get(self._head, conf_path, {})
-        if self._view == "REF":
-            ref_path = ["references"]
-        else:
-            ref_path = ['item', 'bibrecord', 'tail', 'bibliography']
-        self._ref = chained_get(self._json, ref_path, {})
+        if self._view in ('META', 'META_ABS', 'REF', 'FULL'):
+            self._json = self._json['abstracts-retrieval-response']
+            self._head = chained_get(self._json, ["item", "bibrecord", "head"], {})
+            conf_path = ['source', 'additional-srcinfo', 'conferenceinfo', 'confevent']
+            self._confevent = chained_get(self._head, conf_path, {})
+            if self._view == "REF":
+                ref_path = ["references"]
+            else:
+                ref_path = ['item', 'bibrecord', 'tail', 'bibliography']
+            self._ref = chained_get(self._json, ref_path, {})            
```

Comment:
Since you showed that `self._json` exists, let's change this section such that for ENTITLED,  `self._head`, etc. come out as `{}`. That is, use `.get()` to retrieve the paths with optional value. Then the other attributes will properly return `None`, when someone is on the ENTITLED view.

...and same of course for the other classes.